### PR TITLE
refactor(TopNav): remove non-value-add API, remove excess DOM, fix centering

### PR DESCRIPTION
**Pull request by cdaringe**
# problem statement

- `TopNav.Content` has non-value-add API, excess DOM, & bad centering centering

# solution

remove non-value-add API, remove excess DOM, fix centering

# discussion

BREAKING CHANGE: `<TopNav.Content />` props should no longer be used.  Content automatically flows into place.  A `link` prop is added to ease migration, which styles the last `<TopNav.Content link />` to be white, as was previously occurring with the `right` aligned Content piece.

### CODE REVIEW

File: src/components/TopNav/TopNav.examples.md [hunk 1]
```diff
@@ -3,10 +3,10 @@
 ```js
 const logo = require('../../assets/tripwire-logo.png');
 <TopNav id='test-top-nav'>
-  <TopNav.Content className='dummy-test-classname' align="left">
-    <img src={logo} alt="TripWire Logo" width="75" />
+  <TopNav.Content className='dummy-test-classname'>
+    <img src={logo} alt="TripWire Logo" width="75" style={{ marginTop: '4px' }} />
```
Comment (cdaringe) [inline src/components/TopNav/TopNav.examples.md]: the logo has a little tail that makes the largely rectangular logo not symmetric about the horizontally traversing centerline.  that's fine!  this is just an example anyway

File: src/components/TopNav/TopNavContent.jsx [hunk 2]
```diff
@@ -3,22 +3,26 @@ import React from 'react'
 import classnames from 'classnames'
 
 const TopNavContent = props => {
-  const { align, children, className, ...rest } = props
+  const { link, children, className, ...rest } = props
+  const linkClass = link ? 'nav__item--link' : ''
   return (
-    <div className={classnames(`nav__main--${align}`, className)} {...rest}>
-      <div className={`nav__item ${align === 'right' ? 'nav__link' : ''}`}>
```
Comment (cdaringe) [inline src/components/TopNav/TopNavContent.jsx]: trimmed the excess div, and the excess classes.  moved this display mode to flex w/ justify between, as im sure that's 90+% of TopNav use cases

**Comment by wa11-e** [comment]
Woohoo! new surge deployment available for viewing! :tada: [surgereviewpgxmkdvowmvkphe.surge.sh](http://surgereviewpgxmkdvowmvkphe.surge.sh)

File: src/components/TopNav/TopNav.examples.md (hunk 1 above)
Comment (andrewleyva) [reply src/components/TopNav/TopNav.examples.md]: 👍

File: src/components/TopNav/TopNavContent.jsx (hunk 2 above)
Comment (andrewleyva) [reply src/components/TopNav/TopNavContent.jsx]: Cool. Looks good in the CSS below as well.

**Pull request merged**